components/typeahead/typeahead-container.component.ts

```diff
@@ -8,20 +8,20 @@ import { TypeaheadDirective } from './typeahead.directive';
 import { TypeaheadMatch } from './typeahead-match.class';
 
 const bs4 = `
-  <div class="dropdown-menu"
-       [ngStyle]="{top: top, left: left, display: 'block'}"
+  <div #ulElement class="dropdown-menu"
```

Comment:
sorry sorry, my mind interpreted ng2 ids as html ids,
forgive me ;)
forget about this one